### DISCUSSION TITLE
Can both kyuubi param and spark param be passed in param "conf" in /batches API?

### DISCUSSION THREAD

**Discussion by madeirak** [apache/kyuubi#6442] [Q&A] [answered]
This api in kyuubi doc, can Spark and Kyuubi conf be passed in the param "conf" in the red box, or can only Spark conf be passed in?
![image](https://github.com/apache/kyuubi/assets/23215922/a5b71ea7-3373-4cd3-9b2a-da5b4befc650)

**Comment by pan3793** [comment, marked as answer]
Both.
